chsdi/templates/htmlpopup/hiks_dufour_metadata.mako

```diff
@@ -0,0 +1,7 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <tr><td class="cell-left">${_('ch.swisstopo.hiks-dufour.metadata.s_map_number')}</td>         <td>${c['attributes']['s_map_number'] or '-'}</td></tr>
+    <tr><td class="cell-left">${_('ch.swisstopo.hiks-dufour.metadata.s_title_de')}</td>           <td>${c['attributes']['s_title_de'] or '-'}</td></tr>
```

Comment:
I understand, but I would say yes, this way we don't need to go back in the code if these titles are updated.